src/core/model_accessing_API.jl

```diff
@@ -2,10 +2,12 @@ const DictABM = Union{StandardABM{S,A,<:AbstractDict{<:Integer,A}} where {S,A},
                       EventQueueABM{S,A,<:AbstractDict{<:Integer,A}} where {S,A}}
 const VecABM = Union{StandardABM{S,A,<:AbstractVector{A}} where {S,A},
                      EventQueueABM{S,A,<:AbstractVector{A}} where {S,A}}
+const StructABM = Union{StandardABM{S,A,<:StructVector{A}} where {S,A},
```

Comment:
```suggestion
const StructVecABM = Union{StandardABM{S,A,<:StructVector{A}} where {S,A},
```
seems slightly better